src/patient/patientSagas.jsx

```diff
@@ -24,7 +24,8 @@ const createFromReportingRestRep = (restRep) => {
 
   patient.name = {
     givenName: restRep.first_name,
-    familyName: restRep.last_name
+    familyName: restRep.last_name,
+    fullName: `${restRep.first_name} ${restRep.last_name}` 
```

Comment:
@jomadoye this looks good, but one thing I was thinking, if the patient's name is "Tom Jones" and someone types "Jones Tom" will it match?  Probably not a huge blocker but we could have a match string in the format "first_name last_name first_name" that would handle this case?  If we did this, I think we'd make searchFilterFields into a function that the returns that string rather than setting it in patientSagas.js, because it would no longer be "fullName" and not really make sense outside of a search.

Reply:
@jomadoye here's an example of how I used a function instead of an individual field... it allowed me to work around the issue with not being able to specify an array:

https://github.com/PIH/openmrs-pwa-apzu-ic3/blob/master/src/screening/ScreeningQueue.jsx#L37